published/AWS/servers_by_account_and_tags/triggers.json

```diff
@@ -0,0 +1,11 @@
+[ {
+  "text" : "aws_servers_by",
+  "requiresMention" : false,
+  "isRegex" : false,
+  "caseSensitive" : false
+}, {
+  "text" : "aws_servers_by {account_label}  {env_tag_value}",
```

Comment:
The problem that those triggers will never fire since we have other aws behaviors that still the show.